spec.md

```diff
@@ -242,7 +242,7 @@ how platforms might expose these values to their users.
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| parameters | JSON schema object | The schema definition for the input parameters. Schema definitions MUST be valid [JSON Schema draft v4](http://json-schema.org/). It is RECOMMENDED that broker authors explicitly mention the JSON schema draft 4 version, as this will prevent breaking changes when in the future this spec starts supporting future versions of JSON schema with low readability impact. Each input parameter is expressed as a property within a JSON object. Note that UIs MAY not abide by the ordering of input parameters within the schema definition. A service supporting no input parameters SHOULD provide the following schema: `{ "$schema": "http://json-schema.org/draft-04/schema#", "type": "object", "additionalProperties": false }`. |
+| parameters | JSON schema object | The schema definition for the input parameters. Platforms MUST support at least [JSON Schema draft v4](http://json-schema.org/), but SHOULD be prepared to support multiple versions. Schema definitions MUST be [JSON Schema draft v4](http://json-schema.org/) or a later version, and it is RECOMMENDED that broker authors explicitly mention the JSON schema version they are using. Each input parameter is expressed as a property within a JSON object. Note that UIs MAY not abide by the ordering of input parameters within the schema definition. A service supporting no input parameters SHOULD provide the following schema: `{ "$schema": "http://json-schema.org/draft-04/schema#", "type": "object", "additionalProperties": false }`. |
```

Comment:
If no version is specified we should state what the default is or explicitly say that its undefined and up to the platform.

Reply:
I'm fine with how you talk about version support for the Platform. I think we might in the future want to update to call out v6 but looks good for now.

As per Doug's comment add something like... "If no schema version is provided by the broker author, platforms MUST minimally interpret definitions using [JSON Schema draft v4] and if unsuccessful MAY try using later JSON Schema versions."

I would altogether remove the last two statements.
1. We never could depend on ordering in JSON Schema or the more general ordering of JSON properties to mean anything and so feel it is unnecessary. 
2. A statement like "additionalProperties": false is ill advised for backwards compatibility of services that might in the future add properties. It of course makes sense for a service that will never accept parameters but I again would not call it out explicitly and leave it to the author to write their schemas as they see fit.